<launch>
    <node pkg="ut_multirobot_sim" type="simulator_link" name="simulator" cwd="node"
        args="-sim_config config/social_navigation/blind_corner/sim_config.lua" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ut_multirobot_sim)/visualization.rviz" />
</launch>